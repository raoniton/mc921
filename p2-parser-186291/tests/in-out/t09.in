class Main {
    public static void main(String[] args) {
        this.f();
    }

    public int f() {
        int i = 2;
        return i;
    }
}