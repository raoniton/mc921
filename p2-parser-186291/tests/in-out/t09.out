Program:
    ClassDecl: ID(name=Main) @ 1:1
        MainMethodDecl: @ 2:5
            ID: args @ 2:38
            Compound: @ 2:44
                MethodCall: @ 3:9
                    This: @ 3:9
                    ID: f @ 3:14
        MethodDecl: ID(name=f) @ 6:5
            Type: int @ 6:12
            Compound: @ 6:20
                VarDecl: ID(name=i) @ 7:13
                    Type: int @ 7:9
                    Constant: int, 2 @ 7:17
                Return: @ 8:9
                    ID: i @ 8:16
